src/Core/Commander/Providers/WMS_Provider.js

```diff
@@ -145,12 +142,12 @@ WMS_Provider.prototype.getXbilTexture = function(tile, layer, bbox, pitch) {
     var textureCache = this.cache.getRessource(url);
 
     if (textureCache !== undefined) {
-        return Promise.resolve(textureCache ? {
+        return Promise.resolve({
```

Comment:
`null` value for elevation textures is gone with commit `Elevation textures can now have 3 states`.
 It was a special value used for error handling (sort-of).